stripe.go

```diff
@@ -62,6 +69,11 @@ var EnableTelemetry = true
 // Key is the Stripe API key used globally in the binding.
 var Key string
 
+// ClientID is the Stripe Client ID used by default for OAuth requests.
+// Relevant OAuth parameter types can also be initialized with a specific
+// ClientID that will take precidence over this global ClientID.
+var ClientID string
```

Comment:
@remi-stripe Any opinion on having a global `ClientID` versus just requiring that one be passed with parameters? I don't feel too strongly about it, but in general would prefer to avoid globals where possible, so I'd lean toward the latter.

Reply:
We chatted via slack and he agreed with you, so I've removed :)